### DISCUSSION TITLE
Sheduler update: additional Operation Modes "Heat only - Zone related",,possible?

### DISCUSSION THREAD

**Discussion by sabom2d** [edterbak/HeishaMoNR#143] [Ideas]
Hi Ed,

one of my HP (MD9) is connected to Floor heating in zone 2 and radiators in zone 1.
I would like to shedule zone 1 only in the morning and in the evening ( the radiators in the bathroom ) independed from or addionally tozone 2.
Atm I have a timer switch for the pump in zone 1 to establish that,
I tried to take a look at the flow in sheduler, but I do not see through completely, so my question is:
is it possible to have the additional operation mode 9 zone 1 related and mode 10 zone 2 related and additonal mode 11 for both zones?
With this I could save the timer switch and configure everything in your Node-Red flows.
btw. I'm still using Flow Version 23.05 stable

regards

ede

**Comment by edterbak** [comment]
Yrs it is possible to build this functionality.
But... It is not in current solution.

I will take this onboard.

To give some history of this node red flow.
I started purely for myself. 
1 zone. Simpel setup.
Now more and more people are using it. 
Slowly im expanding it, but some of the things are not wel designed (yet) for all other use cases. 

I am working towards 2 zone setup. But it is slow and require major rewriting of the design.

If you give good use requirements,i can design it better. 

It is not built completed and tested soon. Takes a lot of time

**Reply by geduxas** [reply]
I am new to node-red.. but struggling with ability to contribute to this project.. it's seams that flow's are not so friendly to work with versioning control? Or it just me done in some other way?

**Reply by edterbak** [reply]
I struggle with the same.
Working together with branches in github etc. Does not work easily. Maybe i will investigate further. But for now it seems there is no easy robuust solution to that 

If you do know of it, Please inform me as well. This would make mij life easier and wel. 😄

**Reply by geduxas** [reply]
Have you tried this:

https://nodered.org/docs/user-guide/projects/

https://kazuhitoyokoi.medium.com/using-version-control-for-node-red-flows-ce8585df7870

As i told i am new to node-red :D sorry if it's not related

**Comment by edterbak** [comment]
But. Adding more command to the scheduler should not be that difficult. 
I will investigate how to do those command and add it soon

**Comment by edterbak** [comment]
The thing. Restriction, is. That the function rtc is now only designed for 1 zone. 
So this might complicate it.

I will investigate

**Comment by sabom2d** [comment]
Hi Ed,
  

  
Thanks for your explanation.
  
So as I understand you, it's not some easy thing, I could configure myself.
  
Ok, then I will wait for your solution and keep my timer switch solution until then.
  
Atm I have configured it the way that floorheating pump ( Zone 2 ) is running mainly and at 6 in the morning additionally the radiator pump ( Zone 1 ) is activated for 2 hours.
  

  
regards
  

  
ede
  

  

  

  
  
  
  
>
